src/main/java/dk/kb/util/YAMLUtils.java

```diff
@@ -0,0 +1,61 @@
+package dk.kb.util;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class YAMLUtils {
+    
+    /**
+     * Converts the YAML object into a properties object
+     *
+     * @param collection the yaml
+     * @return the yaml as properties
+     */
+    public static Properties toProperties(YAML collection) {
+        Map<String, Object> propertiesMap = toFlatList(collection, null)
+                                                    .collect(Collectors.toMap(
+                                                            tuple2 -> tuple2._1,
+                                                            tuple2 -> tuple2._2
+                                                    ));
+        
+        Properties properties = new Properties();
+        
```

Comment:
Inserting
```
toFlatList(collection, null).forEach(tuple -> properties.put(tuple._1, tuple._2));
```
instead of
```
properties.putAll(propertiesMap);
```
(and removing `propertiesMap`) might be a more intuitive flow than creating the temporary map and adding it all at once.

Having a `forEachLeaf` or similar om `YAML` might be even more intuitive, as it could output the path and the value as separate values, just as the `LinkedHashMap.forEach` does. Thet would make the collecting code
```
collection.forEachLeaf((path, value) -> properties.put(path, value));
```

Reply:
forEachLeaf is stuck as a TODO. This was what I got working...